test/jobs/message.ts

```diff
@@ -0,0 +1,230 @@
+import { describe, it } from 'mocha';
+import { Job, JobStatus } from '../../app/models/job';
+import { hookTransaction } from '../helpers/db';
+import { assert, expect } from 'chai';
+import { baseResultsLimitedMessage } from '../../app/middleware/cmr-granule-locator';
+import { TestTurboService } from '../../app/models/services/turbo-service';
+import DataOperation from '../../app/models/data-operation';
+import env from '../../app/util/env';
+import { v4 as uuid } from 'uuid';
+import { stub } from 'sinon';
+
+/**
+ * A service config to use when building the TestTurboServices.
+ */
+const config = {
+  name: 'first-service',
+  type: { name: 'turbo' },
+  collections: [{ id: 'collection' }],
+  capabilities: {
+    output_formats: ['image/tiff', 'application/x-netcdf4'],
+    subsetting: {
+    },
+  },
+};
+
+/**
+ * Build an operation for the tests.
+ * @returns DataOperation
+ */
+function buildOperation(message: string): DataOperation {
```

Comment:
Is this a function that might be of use in other tests? Should it be moved to a util file for that reason?

Reply:
✅